CHANGELOG.md

```diff
@@ -44,6 +44,10 @@
   [AndrewDMontgomery](https://github.com/andrewdmontgomery)
   [#4875](https://github.com/realm/SwiftLint/pull/4875)
 
+* Adds `grouping` option to the `sorted_imports` rule.  
+  [hiltonc](https://github.com/hiltonc)
+  [#4935](https://github.com/realm/SwiftLint/pull/4935)
```

Comment:
This should be an issue that is addressed by this PR. If there isn't any, we omit this line.